Update Android API docs (#7425)
flutter/website: src/development/tools/sdk/release-notes/supported-platforms.md

|Windows | Windows 7 & above, 64-bit    | All     |

All channels include master, beta,
and stable channels.

## How we define a supported platform

We define three tiers of support for the platforms on
which Flutter runs:

> 1. Supported Google-tested platforms,

Reviewer: ```suggestion
1. Supported Google-tested platforms
```

>    which are platforms the Flutter team at
>    Google tests in continuous integration at every commit.

Reviewer: ```suggestion
  are automatically tested at every commit
  by continuous integration testing.
```

|Web     |Edge 1.2.0            |
|Windows |Windows 10            |

> * Android SDK 20 is covered by testing Android SDK 19. SDK 20 is effectively
>   API 19 with additional support for Android Wear and no new or deprecated API.

Reviewer: ```suggestion
* Passing tests on Android SDK 19 also confers a passing result on SDK 20.
  This is because Android SDK 20 has additional support for Android Wear,
  but otherwise no new or deprecated API.
```
